org.freedesktop.Sdk.Extension.ocaml.yaml

```diff
@@ -0,0 +1,104 @@
+# flatpak-builder --force-clean build-dir org.freedesktop.Sdk.Extension.ocaml.yaml
+# flatpak-builder --user --install --force-clean build-dir org.freedesktop.Sdk.Extension.ocaml.yaml
+id: org.freedesktop.Sdk.Extension.ocaml
+branch: '22.08'
+runtime: org.freedesktop.Sdk
+build-extension: true
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+separate-locales: false
+appstream-compose: false
+build-options:
+  append-path: /usr/lib/sdk/ocaml/bin
+
+modules:
+  - name: bwrap
+    buildsystem: autotools
```

Comment:
why do you need bubblewrap?

Reply:
It is an opam pre-requisite: https://opam.ocaml.org/doc/FAQ.html#Why-does-opam-require-bwrap

Reply:
note that the sandboxing won't work under flatpak, you can't layer two sandboxes, flatpak already uses bubblewrap for the sandboxing it provides. It's fine if it is a build dependency though.

Reply:
You are correct there is a `--disable-sandboxing` flag.
Removed the `bwrap` module.